langchain/chains/conversational_retrieval/base.py

```diff
@@ -132,7 +137,8 @@ def _call(
         else:
             docs = self._get_docs(new_question, inputs)  # type: ignore[call-arg]
         new_inputs = inputs.copy()
-        new_inputs["question"] = new_question
+        if self.use_new_question:
+            new_inputs["question"] = new_question
```

Comment:
should we not generate the new question at all in this case

Reply:
need it for retrieval